phoenix-pherf/cluster/pherf.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Yea, I planned to convert this to python post commit as a new JIRA. Moving the scripts will require some build to Pherf. This pull is already huge so I was trying to limit the functionality changes that could possibly impact phoenix.